go/api-frontend/aaa/authorization.go

```diff
@@ -71,6 +71,7 @@ var pathAdminRolesMap = []adminRoleMapping{
 
 var methodSuffixMap = map[string]string{
 	"GET":    "_READ",
+	"OPTIONS":    "_READ",
```

Comment:
Please run "go fmt" on the files you modify in Golang to keep the formatting consistent and to ensure other folks running it don't start modifying lines they didn't directly touch in future commits